src/view/utils.styl

```diff
@@ -39,3 +39,11 @@ $gap = 8px;
 .u-fieldLong {
   width: 400px !important;
 }
+
+.u-flex {
```

Comment:
Any reason we have to use `!important` here and below?

Reply:
Yes, these are "utility classes". When you use them, you want them to override any conflicting styles the element may have received. If you didn't want them to override conflicting styles, you'd just leave the utility class off the element.

Related reading:

https://davidtheclark.com/on-utility-classes/
https://css-tricks.com/when-using-important-is-the-right-choice/

Precedents in other common CSS libraries: 

https://github.com/suitcss/utils-align/blob/master/lib/align.css
https://github.com/zurb/foundation-sites/blob/develop/scss/prototype/_text-utilities.scss
https://github.com/uikit/uikit/blob/0c3d7058f0f59ca2bcfe300a47edb14d5e10229b/src/less/components/margin.less
https://github.com/twbs/bootstrap/blob/ffaad0a819c6cf1e21d9a9d313673f9d7a260584/scss/mixins/_float.scss

Reply:
I added a small comment to the top of the file in case others have the same question.